packages/teams-js/src/private/associatedApps.ts

```diff
@@ -0,0 +1,237 @@
+import { sendMessageToParentAsync } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { ErrorCode, SdkError, TabInstance } from '../public';
+import { runtime } from '../public/runtime';
+
+// Open questions
+// 1. I've re-used `TabInstance` from the public API, does that contain all of the information you and app developers might need?
+// 2. I didn't see any reason to add a `getTabs` function because `pages.tabs.getTabInstances`. Any reason that won't work for you?
```

Comment:
Does it make sense to deprecate pages.tabs.getTabInstances and use getTabs here instead?

Reply:
Probably not because pages.tabs.getTabInstances is public and the the new functions are all private / 1P only